playbook/app/pb_kits/playbook/pb_hashtag/_hashtag.tsx

```diff
@@ -49,7 +51,10 @@ const Hashtag = (props: HashtagProps) => {
         className={classes}
         id={id}
     >
-      <a href={url}>
+      <a href={url} 
+          rel="noreferrer"
```

Comment:
Since this is an open source project, this really shouldn't be hard-coded here. You can add it as an optional prop then default the value, but we shouldn't dictate this part to everyone.